src/components/filters/Filters.jsx

```diff
@@ -0,0 +1,194 @@
+import { useState } from "react";
+import "./Filters.css";
+
+export const Filters = () => {
+  /** state used to toggle between responsive filter and normal filter **/
+  const [filtersClassName, setFiltersClassName] = useState(
+    "filters-wrapper flex-column"
+  );
+  const checkDeviceWidth = () => {
+    const deviceWidth = window.innerWidth;
+    if (deviceWidth <= 897) {
+      setFiltersClassName("hide-filters");
+    } else {
+      setFiltersClassName("filters-wrapper flex-column");
+    }
+  };
+  window.addEventListener("resize", checkDeviceWidth);
```

Comment:
Suggested to not access DOM directly in react APPS.
And if you absolutely have to. you should put in inside a useEffect.

Reply:
changes done